_sass/_aiml.scss

```diff
@@ -74,33 +73,23 @@
     display: flex;
     justify-content: flex-start;
 }
-.main-content{
-    width: 1000px;
-}
 
-.status-bar{
-    margin-top: 5px;
-    margin-bottom: 20px;;
-    width: 100%;
-    height: 0.5em;
-    border-radius: 10px;
-    border: hidden;
-}
-.current-lesson{
-    background: linear-gradient(to right, $ai-blue 50%, lightgray 50%);
-}
-.ai-blue-background{
+.current-dot{
+    height:10px;
+    width:10px;
     background-color: $ai-blue;
+    border-radius: 50%;
+    display: inline-block;
+    box-shadow: 1px 1px 3px grey;
 }
-.lightgray-background{
-    background-color: lightgray;
-}
+
 #desktop-sidebar{
     width:200px;
-    margin:20px 10px;
+    margin:48px 10px;
     margin-left: calc(50vw - 758px);
     padding:10px 10px;
-    height: calc(100vh - 130px);
+    padding-bottom: 0px;
+    height: calc(100vh - 186px);
```

Comment:
I added comments to it. It basically was the top position plus margin on top and bottom. Would it be better to make variables for all of these (margin, top position) so that the 186px could be repaced with something like `$top-position + 2* $margin-top;`?
I feel like that would be creating a lot of single use variables in the variables.sass file though?